app/vmui/packages/vmui/src/components/Main/Accordion/Accordion.tsx

```diff
@@ -19,6 +19,10 @@ const Accordion: FC<AccordionProps> = ({
   const [isOpen, setIsOpen] = useState(defaultExpanded);
 
   const toggleOpen = () => {
+    const selection = window.getSelection();
+    if (selection && selection.toString()) {
+      return; // Если текст выделен, отменяем выполнение toggle
```

Comment:
Looks like this comment requires translation

Reply:
Yeah, sorry about that! Fixed it now, thanks!